docs/developers/developing_iree/llvm_version_bump.md

```diff
@@ -271,3 +271,39 @@ is less room for error).
 
 Undoing a sequence of cherry-picks is done by integrating to a new upstream
 version, presumably one that includes the commits.
+
+## Tips for reproducing failures locally
```

Comment:
we could also document running the CI pipelines under docker (requires less local setup)